sample_tpl/updater.tpl

```diff
@@ -0,0 +1,116 @@
+
+struct output_struct {
```

Comment:
Add high-level comments indicating its purpose. The final count at the bottom is unclear.